src/components/Read.js

```diff
@@ -4,18 +4,20 @@ import Icon from './Icon'
 
 const Read = ({ letter, handleClick, isWrite }) => {
   return (
-    <div className='ui centered card'>
-      <div className='content'>
-        <p>{letter.content}</p>
-        <span className='right floated icon'>
-          <Icon icon={letter.icon} />
-        </span>
+    <div className='eight wide centered column'>
+      <div className='ui fluid card'>
+        <div className='content'>
+          <p>{letter.content}</p>
```

Comment:
```suggestion
          <p>{content}</p>
```
We should add some destructuring at the top to grab the icon and content